### PR TITLE
style: add CONTRIBUTING and related stuff

### LINKED ISSUE #192: Add a policy doc around staleness

### PR DESCRIPTION
**Pull request by nickatsegment**
It's a start.

Fixes #192

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1 @@
+* nickatsegment eculver
```
Comment (nickatsegment) [inline .github/CODEOWNERS]: Nobody else really seemed to perk up on the idea, but LMK if you're keen and I'll add ya. I'm not sure if we can add org teams, or how that works exactly.

File: .github/stale.yml [hunk 2]
```diff
@@ -7,7 +7,7 @@ exemptLabels:
   - pinned
   - security
 # Label to use when marking an issue as stale
-staleLabel: wontfix
+staleLabel: stale
```
Comment (nickatsegment) [inline .github/stale.yml]: I think `wontfix` sends a dismissive message "won't ever be fixed". I like `stale + closed` --> nobody's working on this. 

`wontfix` means "rejected/out of scope" in my mind

File: .github/stale.yml [hunk 3]
```diff
@@ -7,7 +7,7 @@ exemptLabels:
...
 # Comment to post when marking an issue as stale. Set to `false` to disable
 markComment: >
   This issue has been automatically marked as stale because it has not had
```
Comment (nickatsegment) [inline .github/stale.yml]: Hold tight; I'm going to make this message nicer.

File: .github/stale.yml (hunk 2 above)
Comment (securityclippy) [reply .github/stale.yml]: Agreed.  `Won't fix` means (to me) that this issue has been deemed not-compatible/not an issue while stale just means it doesn't have enough support (yet).   I like this.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+
+An issue/PR being closed doesn't mean that it will never be addressed, just that there currently isn't any intention to do so.
+
+During the grace period, any activity will reset the staleness counter. Generally speaking, this should be a commitment to making progress.
+
+The current staleness policy is defined in [.github/stale.yml](.github/stale.yml).
+
+Stale issues get the `stale` label.
+
+# Labels
```
Comment (nickatsegment) [inline CONTRIBUTING.md]: Hm, some of these don't align with github defaults. https://help.github.com/en/articles/about-labels#using-default-labels Plus we currently have the duplicate `feature request` and `enhancement` labels :/

File: .github/stale.yml [hunk 5]
```diff
@@ -7,11 +7,18 @@ exemptLabels:
...
 markComment: >
-  This issue has been automatically marked as stale because it has not had
-  recent activity. It will be closed if no further activity occurs. Thank you
-  for your contributions.
+  This issue has been automatically marked `stale` because it has not had
+  any activity in the last 60 days. If no further activity occurs within 7
+  days, it will be closed. *Closed does not mean "never"*, just that it has
+  no momentum to get accomplished any time soon.
+
+  See [CONTRIBUTING.md](CONTRIBUTING.md) for more info.
```
Comment (nickatsegment) [inline .github/stale.yml]: I'm not really sure these relative links will work in comments, but worth a shot

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+
+Issues should be triaged within 1 week, where triaging generally means figuring out which type of issue it is and adding labels.
+
+Pull requests (that have had design approval in an issue) should expect responses within 3 days.
+
+If you're finding we aren't abiding by these timelines, feel free to @-mention someone in [CODEOWNERS](.github/CODEOWNERS) to get our attention. If you're the shy type, don't worry that you're bothering us; you're helping us stick the commitments we've made :)
+
+# Staleness
+
+All issues and PRs are subject to staleness after some period of inactivity. An issue/PR going stale indicates there isn't enough interest in getting it resolved. After some grace period, a stale issue/PR will be closed.
```
Comment (majormoses) [inline CONTRIBUTING.md]: I think this makes more sense with pull requests than issues, if there is no actionable item or the submitter does not respond then I agree closing it makes sense. I would like to advocate for keeping valid (enhancement) issues with sufficient information open as this means that it can be found by someone looking to make a contribution and walked over the line. Maybe using some additional status and priority labels can help give us the best of both worlds.

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+- `question`: we'll make an effort to answer your question, but won't guarantee we can solve it.
+
+# Commit messages
+
+We use [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0-beta.3/) to help generate changelogs and do semver releases. We usually do Squash and Merge to PRs, so PR authors are recommended to use the Conventional Commits format in their PR title.
+
+# Anti-contribution
+
+- Obviously, anything that violates our [Code of Conduct](CODE_OF_CONDUCT.md)
+- Noisy comments: "me too!" (:thumbsup: instead) or non-constructive complaining
```
Comment (majormoses) [inline CONTRIBUTING.md]: While I know what your saying I would suggest something along the lines of:
```suggestion
- Noisy comments: "me too!" (instead you should use the :thumbsup: instead on the initial issue or pull request comment as this can be reported on) or non-constructive complaining
```

File: CONTRIBUTING.md (hunk 6 above)
Comment (nickatsegment) [reply CONTRIBUTING.md]: I think this boils down to the fact that Chamber probably isn't big or popular enough to have people roaming by and looking for features to implement because they're bored. I'd guess that almost every feature is added by someone who wants it personally, or for users at their company.

What I *have* seen happen with old feature requests is users stopping by to ask when it's going to be done. I just would rather have it be made clear that feature requests need people to implement them, or at least care for them.

One thing to note is that anybody in the issue can post anything during the staleness period to make it fresh for another 60 days, probably even an emoji. That's a pretty low bar.

File: .github/CODEOWNERS (hunk 1 above)
Comment (nickatsegment) [reply .github/CODEOWNERS]: Yep, agreed. We just need to agree internally on what that team is :)

File: CONTRIBUTING.md (hunk 6 above)
Comment (nickatsegment) [reply CONTRIBUTING.md]: Just going to quote my response from https://github.com/segmentio/chamber/pull/191#issuecomment-472131451

TLDR: duplicates are fine; re-opening when interest renews is fine.

> I think I'd rather have to deal with the slight burden of reconciling duplicate feature requests than the bigger burden of a long list of feature requests that will never get done. An open feature request means that there's some reasonable expectation that someone will pick it up and get it done. If we don't expire feature requests, maybe a user will be able to find an existing feature request easier, but it might give them a false sense of hope that it's in progress. A new issue indicates renewed interest, and perhaps a new contributor :)

File: CONTRIBUTING.md (hunk 4 above)
Comment (nickatsegment) [reply CONTRIBUTING.md]: Ok, I don't want to deal with this at the moment. Going forward, I'll use `enhancement`, and might retcon `feature request`

File: CODE_OF_CONDUCT.md [hunk 8]
```diff
@@ -0,0 +1 @@
+We don't have an official Code of Conduct in place yet. In the meantime, don't be a jerk, and don't discriminate human beings based on irrelevant characteristics. Overuse the :heart: ❤️ emoji.
```
[the remark this answers is not in the document]
Comment (nickatsegment) [reply CODE_OF_CONDUCT.md]: It's already working
